CMakeLists.txt

```diff
@@ -299,16 +299,17 @@ add_custom_command(OUTPUT ${PROJECT_BINARY_DIR}/open62541.h
                    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/tools/amalgamate.py
                            ${GIT_COMMIT_ID} ${CMAKE_CURRENT_BINARY_DIR}/open62541.h ${exported_headers}
                    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/tools/amalgamate.py
-                           ${exported_headers}
-                           ${internal_headers})
```

Comment:
Great stuff!
Last small issue before merging.
I'd keep the exported_headers, internal_headers and lib_sources in the DEPENDS list.
Otherwise the amalgamation will not be rebuild automatically if, say a .c-file, changes during development.